core/src/main/java/uk/gov/gchq/magmacore/database/MagmaCoreRemoteSparqlDatabase.java

```diff
@@ -263,7 +262,7 @@ public List<Thing> findByPredicateIriOnly(final HqdmIri predicateIri) {
      * {@inheritDoc}
      */
     @Override
-    public List<Thing> findByPredicateIriAndStringValue(final IRI predicateIri, final String value) {
+    public List<Thing> findByPredicateIriAndValue(final IRI predicateIri, final Object value) {
         final String query = "SELECT ?s ?p ?o WHERE { ?s ?p ?o.  ?s <" + predicateIri.toString() + "> \"\"\"" + value
```

Comment:
Needs to handle IRIs in the same way as the `MagmaCoreJenaDatabase` class

Reply:
What I was going to say on the call is that this class can only realistically be tested as an integration test since it relies on a network connection to a SPARQL endpoint. I've had problems before where this class fails at runtime even though all the unit tests pass since they only test the in-memory Jena database.

Anyway, this fix _should_ work since it's the same code as the `MagmaCoreJenaDatabase` now.